src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -1,10 +1,12 @@
-@echo off 
+@if "%_echo%" neq "on" echo off
 setlocal
 
-REM 
+REM
 echo Start WCF Self Hosted Service on local machine
 REM
 
+set WCFServiceCleanup=true
```

Comment:
If I have already started the self-host server I think this script should exit immediately.  Can we call the ping powershell script first?  Not blocking for checkin.